R/reorder.R

```diff
@@ -68,6 +69,30 @@ fct_reorder2 <- function(.f, .x, .y, .fun = last2, ..., .desc = TRUE) {
   lvls_reorder(.f, order(summary, decreasing = .desc))
 }
 
+#' Reorder factor levels by sorting along multiple variables
+#' 
+#' @param .f A factor (or character vector).
+#' @param method Passed to \code{base::order()}
+#' @param ordered Passed to \code{\link{fct_inorder}()}
+#' @inheritDotParams base::order -method
+#' @family Reordering
+#' @examples
+#' A <- c(3, 3, 2, 1)
+#' B <- c("A", "B", "C", "D")
+#' fct_reordern(.f = c("A", "B", "C", "D"), A, B)
+#' fct_reordern(.f = c("A", "B", "C", "D"), A, B, decreasing = TRUE)
+#' fct_reordern(.f = c("A", "B", "C", "D"), A, B, decreasing = c(FALSE, TRUE))
+#' @export
+fct_reordern <- function(.f, ..., ordered = NA, method="radix") {
+  f <- check_factor(.f)
+  new_order <- base::order(..., method=method)
```

Comment:
I'd suggest not supply `method` to avoid the R 3.2 issue.

Reply:
Removed.